src/lib/duration/constructor.js

```diff
@@ -33,7 +33,10 @@ export function Duration (duration) {
 
     this._locale = getLocale();
 
-    this._bubble();
+    // _bubble returns a new Duration because immutability,
+    // but here in the constructor we need to update `this` with the new values
+    var bubbled = this._bubble();
+    copyDuration(this, bubbled);
```

Comment:
Yuck.  I don't see a way around this, though, unless we un-publish the `_bubble` method so that it can remain an unwrapped mutator.